puma/var_vs_var.py

```diff
@@ -0,0 +1,409 @@
+"""Variable vs another variable plot"""
+import matplotlib as mpl
+import numpy as np
+from matplotlib.patches import Rectangle
+
+# TODO: fix the import below
+from puma.plot_base import PlotBase, PlotLineObject
+from puma.utils import get_good_colours, get_good_markers, logger
+from puma.utils.histogram import hist_ratio
+
+
+class VarVsVar(PlotLineObject):  # pylint: disable=too-many-instance-attributes
+    """
+    var_vs_eff class storing info about curve and allows to calculate ratio w.r.t other
+    efficiency plots.
+    """
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        x_var_mean: np.ndarray,
```

Comment:
I guess these are not really the mean x-values, but rather the bin midpoints? You could just shorten to `xs` and clarify in the docstring

Reply:
I renamed to `x_var`, since pylint doesn't like `xs`. 
Also I made some clarifications in the docstring, what do you think?

Reply:
Ugh pylint being too aggressive imho... anyways looks good, thanks!